modules/installation/pages/client-requirements.adoc

```diff
@@ -21,15 +21,34 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
 | {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
+| {sles}-ES 8 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
 | {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
+| CentOS 8 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
 | {opensuse} Leap 15.1 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+| {ubuntu} 20.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
 | {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+|===
+
+
+
+When you are setting up your client hardware, you need to ensure you have enough for the operating system and for the workload you want to perform on the client, with these additions for {productname}:
+
+
+[[clients.hw.reqs]]
+[cols="1,1", options="header"]
+.Client Additional Hardware Requirements
+|===
+| Hardware               | Recommended
+| CPU:                   | 1 64-bit CPU core ({x86_64})
+| RAM                    | 2{nbsp}GB
+| Swap space             | 2{nbsp}GB
+| Disk Space:            | 200{nbsp}MB
```

Comment:
I am not 100% happy with this, as it mix minimal values with additional. It give the impression that only our mamagement stack requires 2GB RAM + 2GB Swap.

Suggestion:

```suggestion
.Client Hardware Requirements
|===
| Hardware               | Size
| CPU:                   | minimal 1 64-bit CPU core ({x86_64}) for the system
| RAM                    | minimal 2{nbsp}GB for the system
| Swap space             | recommended 2{nbsp}GB
| Disk Space:            | + 200{nbsp}MB for the management stack
```

Alternative we guess a bit:
- drop CPU (as any value fit)
- RAM: +512MB for the management stack
- drop SWAP space
- Disk Space: 200MB for the management stack

I have no idea if 512MB just for the management stack is sufficient or still over commited.

Reply:
Heh, this is harder than I expected it to be :wink: 
I'll take another look on Monday and see what we can come up with. I don't think there's any rush to get this in.